src/tape/ensemble.py

```diff
@@ -1142,6 +1160,65 @@ def s2n_inter_quartile_range(flux, err):
                 meta=meta,
             )
 
+        # Output standardization
+        if isinstance(batch, EnsembleSeries):
+            if batch.name == self._id_col:
+                batch = batch.rename("result")
+            batch = EnsembleFrame.from_dask_dataframe(batch.to_frame())
+            if len(on) > 1:
+                batch = batch.reset_index()
+                # Need to overwrite the meta manually as the multiindex will be
+                # interpretted by dask as a single "index" column
+                batch._meta = TapeFrame(columns=on + ["result"])
+
+                if by_band:
+                    batch = batch.categorize(self._band_col).pivot_table(
+                        index=on[0], columns=self._band_col, aggfunc="sum"
+                    )
```

Comment:
Could you please explain this part? Why do we use `aggfunc="sum"` here? Why could/should we use `index=on[0]` here?

Reply:
This is a consequence of only having access to `pivot_table` from pandas when it would be ideal to have `pivot` or `unstack`. With the way that batch sets up it's grouping, I don't believe the aggregation here will ever do any work.

Setting the index on "on[0]" lets pivot_table know that the index of the result should be the first grouping key, which seemed like the ideal behavior as the first key is used as main index in the non by-band case.

Reply:
Thank you!